modules/prometheus/k8s_via_kube_state.hcl

```diff
@@ -252,66 +241,53 @@ ingester prometheus_kube_node module {
     "default" : "$input{using}"
   }
 
-  gauge "total_cpu_for_scheduling" {
+  gauge "disk_pressure" {
     unit = "count"
 
-    source prometheus "total_cpu_for_scheduling" {
-      query = "label_set(sum by (cluster, node) (kube_node_status_allocatable{resource='cpu', unit='core'}) - sum by (cluster, node) (kube_pod_container_resource_limits{resource='cpu', unit='core'}), 'cluster', '$input{cluster}')"
-      join_on = {
-        "$output{cluster}" = "$input{cluster}"
-      }
-    }
-  }
-
-  gauge "total_memory_for_scheduling" {
-    unit = "bytes"
-
-    source prometheus "total_memory_for_scheduling" {
-      query = "label_set(sum by (cluster, node) (kube_node_status_allocatable{resource='memory', unit='byte'}) - sum by (cluster, node) (kube_pod_container_resource_limits{resource='memory', unit='byte'}), 'cluster', '$input{cluster}')"
-      join_on = {
-        "$output{cluster}" = "$input{cluster}"
-      }
-    }
-  }
-
-  gauge "out_of_pods" {
-    unit = "count"
-
-    source prometheus "out_of_pods" {
-      query = "label_set(sum by (cluster, node) (kube_node_spec_unschedulable{}), 'cluster', '$input{cluster}')"
+    source prometheus "disk_pressure" {
+      query = <<EOF
+      sum by (cluster, node)(kube_node_status_condition{condition='DiskPressure', status='true'})/
+      sum by (cluster, node)(kube_node_status_condition{condition='DiskPressure'})
+      EOF
       join_on = {
         "$output{cluster}" = "$input{cluster}"
       }
     }
   }
 
-  gauge "high_disk_pressure" {
+  gauge "pid_pressure" {
     unit = "count"
 
-    source prometheus "high_disk_pressure" {
-      query = "label_set(sum by (cluster, node) (kube_node_status_condition{condition='DiskPressure', status='true'}), 'cluster', '$input{cluster}')"
+    source prometheus "pid_pressure" {
+      query = <<EOF
+      sum by (cluster, node)(kube_node_status_condition{condition='PIDPressure', status='true'})/
+      sum by (cluster, node)(kube_node_status_condition{condition='PIDPressure'})
+      EOF
       join_on = {
         "$output{cluster}" = "$input{cluster}"
       }
     }
   }
 
-  gauge "high_memory_pressure" {
+  gauge "memory_pressure" {
     unit = "count"
 
-    source prometheus "high_memory_pressure" {
-      query = "label_set(sum by (cluster, node) (kube_node_status_condition{condition='MemoryPressure', status='true'}), 'cluster', '$input{cluster}')"
+    source prometheus "memory_pressure" {
+      query = <<EOF
+      sum by (cluster, node)(kube_node_status_condition{condition='MemoryPressure', status='true'})/
```

Comment:
All metric queries k8s_via_kube_state.hcl - should have `label_set` block on them. Applies to rest of the queries also.

Reply:
aah thought as much. Cool will fix.

Reply:
@saurabh-hirani In the same reference for `prometheus_kube_container` ingester the queries looks a bit different. Can you check and let me know how to proceed?

Reply:
Did not follow. Can you add more details or let us sync up once?